### PR TITLE
Resolves camel-14731. Rearrange sources to use Antora structure more effectively

### PR DESCRIPTION
**Pull request by djencks**
This renames the playbook to current standard name "antora-playbook.yml" and adjusts the sources to find the components as they are in the other 3 PRs in this set of tasks.  This playbook will only work after the other three PRs are merged.  Before that, use the antora-playbook.yml from https://github.com/djencks/camel-website.git branch issue-14698-rearrange-adocs.  Because of this, this PR is untested.

### CODE REVIEW

**Comment by oscerd** [comment]
@djencks this doesn't seem to work actually 

[2020-03-18T18:44:20.210Z] + cd /home/jenkins/jenkins-slave/workspace/Camel.website_PR-230/camel-website
[2020-03-18T18:44:20.210Z] + yarn --non-interactive --frozen-lockfile build
[2020-03-18T18:44:20.795Z] yarn run v1.22.0
[2020-03-18T18:44:20.795Z] $ run-s build:*
[2020-03-18T18:44:21.382Z] $ antora --require ./menu.js --clean --fetch antora-playbook.yml
[2020-03-18T18:45:58.129Z] error: core/camel-core-languages/src/main/docs/antora.yml not found in https://github.com/djencks/camel.git [ref: remotes/origin/master]
[2020-03-18T18:45:58.130Z] Add the --stacktrace option to see the cause.
[2020-03-18T18:45:58.130Z] error Command failed with exit code 1.
[2020-03-18T18:45:58.130Z] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[2020-03-18T18:45:58.130Z] ERROR: "build:antora" exited with 1.
[2020-03-18T18:45:58.130Z] error Command failed with exit code 1.
[2020-03-18T18:45:58.130Z] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

I suppose this is because it is pointing to your fork of camel and it's not up-to-date

File: antora-playbook.yml [hunk 1]
```diff
@@ -1,14 +1,23 @@
 site:
   title: Apache Camel
   url: https://camel.apache.org
-  start_page: manual::index.adoc
+  start_page: latest@manual::index.adoc
 
 content:
   sources:
-    - url: git@github.com:apache/camel.git
+    - url: https://github.com/djencks/camel.git
```
Comment (oscerd) [inline antora-playbook.yml]: We need to change the repository to have a full test.

**Comment by oscerd** [comment]
ok to test

File: antora-playbook.yml (hunk 1 above)
Comment (djencks) [reply antora-playbook.yml]: OOPS!  I pushed a fix that works OK for me locally.

BTW when I run mvn clean install and then git status there are an enormous number of files deleted from static.  Is this expected?

**Comment by oscerd** [comment]
Started the job by hand.

File: antora-playbook.yml (hunk 1 above)
Comment (oscerd) [reply antora-playbook.yml]: No, this shouldn't happen. I can reproduce it locally and I don't know why. The schemas must be there as all the static content.

Comment (oscerd) [reply antora-playbook.yml]: So please don't push that part and otherwise open an issue for this, so we can double check why

Comment (oscerd) [reply antora-playbook.yml]: fixed here https://github.com/apache/camel-website/pull/240

**Comment by oscerd** [comment]
The preview is here https://builds.apache.org/job/Camel.website/job/PR-230/Preview @davsclaus @djencks @zregvart

**Comment by oscerd** [comment]
Looks good to me. Waiting for @davsclaus and @zregvart comment

**Comment by davsclaus** [comment]
+1

**Comment by oscerd** [comment]
Eventually we can rework this, let me try to deploy the new site. Thanks @djencks

**Pull request merged**